devscripts/pyinst.py

```diff
@@ -27,7 +27,7 @@
 exec(compile(open('youtube_dlc/version.py').read(), 'youtube_dlc/version.py', 'exec'))
 VERSION = locals()['__version__']
 
-VERSION_LIST = VERSION.replace('-', '.').split('.')
```

Comment:
Why was this changed? This is for if there are two releases in 1 day.

Reply:
In youtube-dlc, the version number was of the form `2021.02.09-1`. But I changed it to `2021.02.09.1`.

`.replace('-', '.')` was needed during the transition so that the new code can parse the old version number. But it is no longer needed